app.js

```diff
@@ -44,25 +47,74 @@ app.Views.TodoAppView = Backbone.View.extend({
         this.todoContent.append(this.subviews.todoListView.render());
 
         return this.$el;
+    },
+    createTodoOnEnter: function(e) {
+        const title = this.newTodoField.val().trim();
+        if (e.which == 13 && title != "") {
+
+            this.subviews.todoListView.collection.add(new app.Models.TodoItem({
+                title: title
```

Comment:
in ES6, when the property name and the value you are assigning are named the same, you can just type it once...

```
{
    title
}
```

Reply:
Cool